dluhc-component-library/src/stories/SiteSelection.stories.tsx

```diff
@@ -52,6 +52,17 @@ export const Default = {
           type: "string",
           title: "Your name",
         },
+        age: {
+          type: "number",
+          title: "Age in years",
+          step: "1",
+          min: "18",
```

Comment:
Looking at the JsonSchema docs these should be numbers not strings.

Also annoyingly there are two types. "number" and "integer". Lets ignore this for now though